src/Schema/Index.php

```diff
@@ -334,6 +351,11 @@ public function getOptions()
         return $this->options;
     }
 
+    public static function isFunctionalIndex(string $name): bool
+    {
+        return $name[0] === '(' && $name[mb_strlen($name) - 1] === ')';
```

Comment:
Please use `substr()` to fetch the last character of `$name`.

Reply:
Theoretically, the expression can contain non-Latin characters. For example, this is valid statement
```
CREATE FUNCTION АБ⍅ (s CHAR(20))
    RETURNS CHAR(50) DETERMINISTIC
	RETURN CONCAT('Hello, ',s,'!');
```

Reply:
Replaced by `mb_substr`

Reply:
Yes, but `)` iself is an ASCII character. `substr()` is safe here. We don't want to depend on ext-mbstring to be present.

Reply:
Done